teuthology/task/install/__init__.py

```diff
@@ -135,6 +135,8 @@ def remove_packages(ctx, config, pkgs):
     with parallel() as p:
         for remote in ctx.cluster.remotes.iterkeys():
             system_type = teuthology.get_system_type(remote)
+            if remote.os.name in ['opensuse', 'sle']:
+                remote.run(args='zypper lr')
```

Comment:
I find `zypper lr -upEP` to be more informative.

Reply:
Oops, those two lines should not be present here, forgot to remove it after debugging.